easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -127,9 +126,22 @@ def __init__(self, *args, **kwargs):
             self.install_cmd_extra = '.'
 
         else:
-            self.install_cmd = "python setup.py install"
+            self.install_cmd = "python %(python_opts)s setup.py install"
             self.install_cmd_extra = None
 
+        self.python_opts = []
+
+    def set_python_opts(self):
+        """Set options to pass to 'python'."""
+        # use python -S when using system Python, to avoid that Python packages installed system-wide are picked up
```

Comment:
why would you be using the system python in the first place?shouldn't this be an error?